ui/tests/acceptance/secrets/backend/kv/kv-v2-workflow-delete-test.js

```diff
@@ -38,12 +27,12 @@ module('Acceptance | kv-v2 workflow | delete, undelete, destroy', function (hook
 
   module('admin persona', function (hooks) {
     hooks.beforeEach(async function () {
-      const token = await runCmd([
-        createPolicyCmd('admin', adminPolicy(this.backend)),
-        createTokenCmd('admin'),
-      ]);
+      const token = await runCmd(tokenWithPolicyCmd('admin', personas.admin(this.backend)));
```

Comment:
Replaced ad-hoc `createPolicy` calls with this persona, which will guarantee we're always testing with similar policies when using these personas